projects/storefrontlib/src/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -16,6 +19,79 @@ export class PaginationComponent {
   @Input() pagination: PaginationModel;
   @Output() viewPageEvent: EventEmitter<number> = new EventEmitter<number>();
 
+  getPagePrevious(): number {
```

Comment:
logic (naming mostly) doesn't make sense. you want to get pagePrevious, but you return currentPage?